### ISSUE TITLE
iOS production release crash

### ISSUE THREAD

**Issue by itemir** [becvert/cordova-plugin-zeroconf#10]
An iOS build using Ionic (not sure applicable to other frameworks but possibly) works in debug mode but it causes an immediate application crash in the production/release mode. 

Problem can be solved by adding "@executable_path/Frameworks" to Runpath Search Paths for Release (it is already there for Debug).

There used to be instructions for this in the README before, but they were removed by https://github.com/becvert/cordova-plugin-zeroconf/commit/f5d2a779826d0a2540b1f08b45c4d7fe582e7d29 commit.

**Comment by becvert** [comment]
Hi @itemir,
the new dependency on [cordova-plugin-add-swift-support](https://github.com/akofman/cordova-plugin-add-swift-support) takes care of the Swift config.
It should have worked for both debug and release modes.
Have a look over there.
Thanks

**Comment by itemir** [comment]
Thanks @becvert. I already did have cordova-plugin-add-swift-support, I think it is a dependency for the last version so it gets loaded automatically anyways. 

Even with cordova-plugin-add-swift-support, the problem happens. I only use cordova-plugin-zeroconf with Ionic, not sure if applicable to other installation scenarios. 

Feel free to close the ticket. I already have a manual solution. I created it to bring awareness.
